tools/gce-usage-log/.gitignore

```diff
@@ -0,0 +1 @@
+target/
```

Comment:
only did this cuz when i pulled it down to run the tests I accidentally almost committed it :)